chaos_genius/core/utils/data_loader.py

```diff
@@ -105,14 +119,18 @@ def _convert_date_to_string(self, date: date, offset: str):
         # we shouldn't need to take offset as a string, but rather
         # take in a pytz timezone and skip using strings.
         date = date.strftime("%Y-%m-%d")
-        date += f"T00:00:00{offset}"
+        date += self.db_connection.SQL_DATE_TO_DATETIME_FORMAT.format(offset)
         if not self.kpi_info.get("timezone_aware"):
             date = (
-                pd.Timestamp(datetime.strptime(date, "%Y-%m-%dT%H:%M:%S%z"))
+                pd.Timestamp(
+                    datetime.strptime(
+                        date, self.db_connection.SQL_STRPTIME_FORMAT
+                    )
+                )
                 .tz_convert(self.connection_info["database_timezone"])
                 .tz_localize(None)
                 # TODO: We should also use date.isoformat() here
-                .strftime("%Y-%m-%dT%H:%M:%S")
+                .strftime(self.db_connection.SQL_STRFTIME_FORMAT)
```

Comment:
Let's have utils functions in the `datetime_helper.py` for this.

Reply:
We're going to replace the function as soon as we remove 3 letter timezones. I don't think we need to create a util function for now and then remove it once it's done.

Reply:
Please add the TODO in that case.

Reply:
There's already a TODO for it in [line 118](https://github.com/chaos-genius/chaos_genius/blob/085b7de7df5d94d724db55c83f98c2a10389d56a/chaos_genius/core/utils/data_loader.py#L118).